src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,108 @@
-import React from 'react';
+import React, {
+  Dispatch, SetStateAction, useEffect, useState,
+} from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  moviesList: Movie[],
+  onMovieAdd: Dispatch<SetStateAction<Movie[]>>
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ moviesList, onMovieAdd }) => {
+  const [movieTitle, setMovieTitle] = useState('');
+  const [isMovieFound, setIsMovieFound] = useState<null | boolean>(null);
+  const [foundMovie, setFoundMovie] = useState<null | Movie>(null);
+
+  useEffect(() => {
+    setIsMovieFound(null);
+  }, [movieTitle]);
+
+  const fetchMovie = async () => {
+    const movie = await getMovie(movieTitle);
+
+    if (movie.Response === 'True') {
+      setIsMovieFound(true);
+      setFoundMovie(movie);
+    } else {
+      setIsMovieFound(false);
+      setFoundMovie(null);
+    }
+  };
+
+  const addMovie = () => {
+    if (!foundMovie || moviesList.includes(foundMovie)) {
+      return;
+    }
```

Comment:
```suggestion
    if (!foundMovie || moviesList.includes(foundMovie)) {
      return;
    }
```
did you compare object links?